dissect/target/plugins/general/network.py

```diff
@@ -0,0 +1,77 @@
+from functools import lru_cache
+from typing import Iterator, Union
+
+from dissect.target.helpers.record import (
+    MacInterfaceRecord,
+    TargetRecordDescriptor,
+    UnixInterfaceRecord,
+    WindowsInterfaceRecord,
+)
+from dissect.target.plugin import Plugin, Record, export, internal
+from dissect.target.target import Target
+
+# Using this notation as it didn't like the pipe symbol
```

Comment:
```suggestion
```
Don't think a comment in the code is necessary for this. It's because we're being naughty boys and using variables as type hints.